src/composables/auth.ts

```diff
@@ -44,6 +44,7 @@ import { useModals } from './modals';
 import { useStorageRef } from './storageRef';
 
 const CHECK_FOR_SESSION_KEY_INTERVAL = 5000;
+let expirationTimeout: NodeJS.Timeout;
```

Comment:
This can be placed inside of the composable as it is using the `createCustomScopedComposable`.